class.application-passwords.php

```diff
@@ -340,10 +340,14 @@ public static function show_user_profile( $user ) {
 		wp_enqueue_style( 'application-passwords-css', plugin_dir_url( __FILE__ ) . 'application-passwords.css', array() );
 		wp_enqueue_script( 'application-passwords-js', plugin_dir_url( __FILE__ ) . 'application-passwords.js', array() );
 		wp_localize_script( 'application-passwords-js', 'appPass', array(
-			'root'       => esc_url_raw( rest_url() ),
-			'namespace'  => '2fa/v1',
-			'nonce'      => wp_create_nonce( 'wp_rest' ),
-			'user_id'    => $user->ID,
+			'root'       		=> esc_url_raw( rest_url() ),
+			'namespace'  		=> '2fa/v1',
+			'nonce'      		=> wp_create_nonce( 'wp_rest' ),
+			'user_id'    		=> $user->ID,
+			'translations'	=> array(
+				'revoke_password'				=> esc_attr__( 'Are you sure you want to revoke this password? This action cannot be undone.' ),
```

Comment:
It's probably because of how many spaces a tab may equal.  Tabs to indent,
spaces to align.

On Tue, Feb 9, 2016 at 4:50 PM, Kyle Benk <notifications@github.com> wrote:

> In class.application-passwords.php
> <https://github.com/georgestephanis/application-passwords/pull/23#discussion_r52379753>
> :
>
> > @@ -340,10 +340,14 @@ public static function show_user_profile( $user ) {
> >  		wp_enqueue_style( 'application-passwords-css', plugin_dir_url( __FILE__ ) . 'application-passwords.css', array() );
> >  		wp_enqueue_script( 'application-passwords-js', plugin_dir_url( __FILE__ ) . 'application-passwords.js', array() );
> >  		wp_localize_script( 'application-passwords-js', 'appPass', array(
> > -			'root'       => esc_url_raw( rest_url() ),
> > -			'namespace'  => '2fa/v1',
> > -			'nonce'      => wp_create_nonce( 'wp_rest' ),
> > -			'user_id'    => $user->ID,
> > +			'root'       		=> esc_url_raw( rest_url() ),
> > +			'namespace'  		=> '2fa/v1',
> > +			'nonce'      		=> wp_create_nonce( 'wp_rest' ),
> > +			'user_id'    		=> $user->ID,
> > +			'translations'	=> array(
> > +				'revoke_password'				=> esc_attr__( 'Are you sure you want to revoke this password? This action cannot be undone.' ),
>
> I usually use tabs to indent and for some reason it looks fine in my
> editor but then appears to be mis-aligned in Github, I am not sure as to
> why.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/georgestephanis/application-passwords/pull/23/files#r52379753>
> .
>